packages/editor/src/components/EditorContainer.css

```diff
@@ -1,76 +1,64 @@
-/* this is basically style for the editor page*/
-/* we can drill down furthur if needed*/
+/** references https://github.com/ticlo/rc-dock/blob/master/dist/rc-dock.css */
```

Comment:
renamed `Editor2Container.css` as `EditorContainer.css` and moved the contents from `Editor2Container.css` to here